### ISSUE TITLE
tl_PH Localization

### ISSUE THREAD

**Issue by cordeta** [mozilla/olm-whitepaper#32]
Hi , if you may i would like to make a Tagalog Localization of your whitepaper 

Hoping for your response 

Thanks !

**Comment by chadsansing** [comment]
@cordeta, that would be wonderful. Thank you for suggesting and making this contribution. 

I wonder if other contributors might help you or be interested in other localization projects during the [Global Sprint](https://medium.com/read-write-participate/preparing-for-mozillas-2018-global-sprint-2a37dc3372c4), as well.

**Comment by stephwright** [comment]
That's fantastic @cordeta ! Thank you for offering! And I think that's a great idea @chadsansing re: the Global Sprint!

**Comment by cordeta** [comment]
Yeah sure, I have some of my friends that might be of help as well.  

Just a quick intro, I am part of the Filipino Utopian translators. btw you should consider making an il8n folder so that future translators could easily see it. If you have some files that you want to be localized, we would be happy to work with it.

Cheers ! :100:

**Comment by chadsansing** [comment]
Will do, @cordeta!

**Comment by cordeta** [comment]
Thanks @chadsansing  Just let me know when will i start so that i could fork this one and make it as our staging repo for translation as I will personally check the works of my team.

**Comment by cordeta** [comment]
Hi @chadsansing 

may i know when shall i start ?
Thanks

**Comment by chadsansing** [comment]
I can create a folder Monday or Tuesday.

**Comment by cordeta** [comment]
Got it,

Anyways have a great day!

**Comment by chadsansing** [comment]
@cordeta, started an il8n folder here:

https://github.com/mozilla/olm-whitepaper/tree/master/il8n

Let me know what else might be helpful to include or format differently?

**Comment by cordeta** [comment]
Ohw Thank You!, now that you have made the folder, can I make a tl_PH folder inside [tl_PH stands for Tagalog - Philippines]? so that I can put all the translated files there so it won't be messy if other translators would come in. pretty sure all the translators of different languages will also do the same thing ;) I'll just fork this one and I'll make a PR if I'm finished with it 

Cheers

**Comment by chadsansing** [comment]
That sounds right - thanks, @cordeta!

**Comment by cordeta** [comment]
always welcome , thanks you for entrusting me this one

**Comment by chadsansing** [comment]
@cordeta - just a note: we are changing the name of the document.

Old: Open Leadership Map
New: Open Leadership Framework

Please let me know if you have any questions.

**Comment by drashti4** [comment]
Hi there, I would like to work on this issue. [https://github.com/mozilla/olm-whitepaper/tree/master/il8n](https://github.com/mozilla/olm-whitepaper/tree/master/il8n) isn't working.

**Comment by chadsansing** [comment]
Try https://github.com/mozilla/open-leadership-framework/tree/master/il8n, @drashti4? We renamed the repo a while back - apologies for any confusion!

**Comment by drashti4** [comment]
No worried @chadsansing Can we have some beginner friendly issue with labels? I know this is really cool project and many people can contribute if we give right directions.  :smile_cat:

**Comment by chadsansing** [comment]
Surely. Suggestions? I can come back to this on Monday, June 18th, and add the issues and labels then - work meeting all next week.

**Comment by drashti4** [comment]
If we planning to localize this framework, Which tool we're using or going to use?
